sdcm/fill_db_data.py

```diff
@@ -3107,3 +3107,16 @@ def verify_db_data(self):
             # override driver consistency level
             session.default_consistency_level = ConsistencyLevel.QUORUM
             self.run_db_queries(session, session.default_fetch_size)
+
+    def paged_query(self):
+        # Prepare connection
+        node = self.db_cluster.nodes[0]
+        with self.db_cluster.cql_connection_patient(node, keyspace='keyspace_complex') as session:
+            statement = 'SELECT * FROM user_with_ck'
+            full_query_res = session.execute(statement)
+            full_res = self.fetch_all_rows(session=session, default_fetch_size=1000, statement=statement)
+            self.assertEqual(len(full_query_res), len(full_res), "Both results should have same lengh but they differ")
+            limit = 10000
+            limit_res = self.fetch_all_rows(session=session, default_fetch_size=1000,
+                                            statement=f'{statement} LIMIT {limit}')
+            self.assertEqual(len(limit_res), limit, f"Result was expected to be same as 'LIMIT'")
```

Comment:
What do you check here?

Reply:
adding to the `paged queries` a `limit`, that must be respected nonetheless the amount of pages needed to be read to attend that limit